docs/ATTRACTOR_MEMORY_EXPERIMENT.md

```
@@ -0,0 +1,145 @@
+Every diagonal value must be zero.
+
+Recall must use the weight matrix rather than a filename, database key,
+pattern identifier, or lookup index.
+
+## Settling rule
+
+Use deterministic asynchronous neuron updates.
+
+For each neuron:
+
+field = sum(W[i][j] * state[j])
```

markdownlint-cli2 (0.23.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "j"

(MD052, reference-links-images)